src/commands/Find.thy

```diff
@@ -0,0 +1,75 @@
+theory Find
+  imports Main
+begin
+
+section \<open>Find theorems with @{command find_theorems}\<close>
+text \<open>
+  Theorems can be searched by name where @{verbatim *} wildcards are allowed.
+  One of the theorems found with the query below is @{thm gauss_sum_nat}.
+\<close>
+find_theorems name: "gau*"
+
+text \<open>
```

Comment:
Possible additions:
- searching by pattern and name can also be combined
- to look for infix operators like `<`, we either need to use the form `_ < _` or `(<)`